hicommonwealth/commonwealth
# #6847 - Remove cruft around simultaneous creation of topic and thread

This is a follow-up to #6495, wherein we removed the ability to create a thread and topic at the same time from the UI. This PR removes that functionality on the backend, as well. 

## Link to Issue
Closes: #6847

## Description of Changes
- removes references to a topic being featured in a post/thread and the associated thread

## Test Plan
- navigate to the Topics page under the Admin Capabilities 
- ensure that you can still create and edit topics

## Follow-Up
We'll need to remove the attributes around a topic being featured in a thread from the database
